Source/UI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -197,6 +186,21 @@ final class DefaultBillingFormViewModel: BillingFormViewModel {
 
         updatedRow = textField.tag
     }
+
+    private func notifyContentChangeToDelegate() {
+        let hasErrorValue = errorFlagOfCellType.isEmpty || errorFlagOfCellType.values.allSatisfy({$0})
+        guard !hasErrorValue else {
+            editDelegate?.didFinishEditingBillingForm(successfully: false)
+            return
+        }
+
+        let areAllFieldFulfilled = style.cells.first(where: {
+          guard let style = $0.style, style.isMandatory else { return false }
+          return textValueOfCellType[$0.index]?.isEmpty != false
+        }) == nil
+
+        editDelegate?.didFinishEditingBillingForm(successfully: areAllFieldFulfilled)
+    }
```

Comment:
this is duplicate from another PR. I think, you can share same logic to the other PR and i will test it.

Reply:
Actually this is what testing is for, to flag issues in the code. And I think this PR did well to catch the boolean difference, as well as the fact the Country update itself was not triggering a validation recheck.

Also as mentioned in the other PR, the solution proposed there is to use inefficient and verbose syntax, not to actually fix a simple boolean comparison.

Reply:
Let's wait for other PR to be merged then this part will be solved.

Reply:
@alex-ioja-yang-cko As i understand this small change might conflict with other PR what @ehab-al-cko has already worked on, So would be worth discussing the changes before or waiting for other PRs to be merged. Will save time and effort.

Reply:
i think, if this PR is merged then no need for other PR.